docs/configuring-polaris-for-production.md

```diff
@@ -56,7 +56,7 @@ Be sure to secure your metastore backend since it will be storing credentials an
 
 ### Configuring EclipseLink
 
-To use [EclipseLink](https://eclipse.dev/eclipselink/) for metastore management, specify the configuration `metaStoreManager.conf-file` to point to an [EclipseLink `persistence.xml` file](https://eclipse.dev/eclipselink/documentation/2.5/solutions/testingjpa002.htm). This file, local to the Polaris service, will contain information on what database to use for metastore management and how to connect to it.
+To use [EclipseLink](https://eclipse.dev/eclipselink/) for metastore management, specify the configuration `metaStoreManager.conf-file` to point to an [EclipseLink `persistence.xml` file](https://eclipse.dev/eclipselink/documentation/4.0/solutions/solutions.html#TESTINGJPA002). This file, local to the Polaris service, contains details of the database used for metastore management and the connection settings. For more information, refer to [metastore documentation](./metastores.md) for details.
```

Comment:
I don't think people will understand the summary, especially for the first-time reader. I feel It even causes more confusion. How about just this?
```
Check [metastore documentation](./metastores.md) for instructions.
```

Reply:
I think we need to provide some summary here to give some high level information and can look at the separate page if needed to understand more.

Reply:
The point is that developer will always have to click into the detailed page for instructions. There is a rare chance that they can do anything by just looking at the summary. Moreover, we provided 3 links in the summary here, there is a good chance that reader will click first two links, but still lost. What they really need is the third link(./metastores.md).

Reply:
That makes sense. I will remove the first two links so the users can check out the separate page metastores.md.